.gitignore

```diff
@@ -2,3 +2,6 @@ target
 **/target
 **/corpus
 **/artifacts
+
+.idea/
+.vscode/
```

Comment:
generally prefer to keep these in your personal ~/.gitignore. Not a huge deal, but in interest of keeping the PR small and focused...
```suggestion
```

Reply:
Right good point.